plugins/modules/info.py

```diff
@@ -184,13 +185,38 @@
     description:
     - Contains dictionary of query parameters for specific I(gather_subset).
     - Applicable to C(alert_rules), C(event_group), C(event_channels) and C(filesystem).
+    - If C(writable_snapshot) is passed as I(gather_subset), if I(wspath) is given,
```

Comment:
should we keep the sub options, for event channels and rules sub options were not added.
@Bhavneet-Sharma @meenakshidembi691 FYI , please comment.